src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -139,49 +144,64 @@ mutable struct AdditiveRungeKutta{
 
         # The code throughout assumes SDIRK implicit tableau so we assert that
         # here.
-        for is in 2:Nstages
-            @assert RKA_implicit[is, is] ≈ RKA_implicit[2, 2]
-        end
+        # for is in 2:Nstages
+        #     @assert RKA_implicit[is, is] ≈ RKA_implicit[2, 2]
+        # end
```

Comment:
Remove dead code? Or can we run the check is the right variant is set? (e.g., does `LowStorageVariant` imply `SDIRK`?)

Reply:
Could also check that if `LowStorageVariant ` that `RKB_implicit == RKB_explicit` and `RKC_implicit == RKC_explicit`

Reply:
yes, we removed the dead code. Will push soon.